docs/public-networks/get-started/connect/mainnet.md

```diff
@@ -0,0 +1,144 @@
+---
+description: How to connect to Mainnet
+---
+
+# Connect to Mainnet
+
+!!! important
+
+    [The Merge](../../concepts/the-merge.md) was executed on **September 15, 2022**.
+    Ethereum is now a [proof of stake](../../concepts/proof-of-stake.md) network, and a full Ethereum
+    node requires both [an execution client and a consensus client](../../concepts/the-merge.md#execution-and-consensus-clients).
+
+Run Besu as an [execution client](../../concepts/the-merge.md#execution-clients) with any consensus
+client on Ethereum Mainnet.
+
+If you're using [Teku] as a consensus client, you can follow the
+[Besu and Teku Mainnet tutorial](../../tutorials/besu-teku-mainnet.md).
+
+## Prerequisites
+
+- [Besu installed](../install/binary-distribution.md).
+- A consensus client installed.
+  For example, [Teku](https://docs.teku.consensys.net/en/latest/).
+
+## Steps
+
+### 1. Generate the shared secret
+
+Run the following command:
+
+```bash
+openssl rand -hex 32 | tr -d "\n" > jwtsecret.hex
+```
+
+You will specify `jwtsecret.hex` when starting both Besu and the consensus client.
+This is a shared JWT secret the clients use to authenticate each other when using the
+[Engine API](../../how-to/use-engine-api.md).
+
+### 2. Start Besu
+
+Run the following command or specify the options in a
+[configuration file](../../how-to/configuration-file.md):
+
+```bash
+besu \
+  --sync-mode=X_SNAP           \
+  --data-storage-format=BONSAI \
+  --rpc-http-enabled=true      \
+  --rpc-http-host=0.0.0.0      \
+  --rpc-ws-enabled=true        \
+  --rpc-ws-host=0.0.0.0        \
+  --host-allowlist=<IP of Besu node>,127.0.0.1,localhost        \
+  --engine-host-allowlist=<IP of Besu node>,127.0.0.1,localhost \
+  --Xmerge-support=true        \
```

Comment:
I assume that soon this will be the default?

Reply:
Probably, but I haven't been able to get more information on the configuration options from the Besu team. All Discord users who have shared their configuration have set this to true.